packages/lab/src/overlay/OverlayPanelContent.tsx

```diff
@@ -1,13 +1,24 @@
 import { useWindow } from "@salt-ds/window";
 import { useComponentCssInjection } from "@salt-ds/styles";
 import { makePrefixer } from "@salt-ds/core";
-import { ComponentPropsWithoutRef } from "react";
+import { ReactNode, ComponentPropsWithoutRef, forwardRef } from "react";
 import clsx from "clsx";
 import overlayPanelContentCss from "./OverlayPanelContent.css";
 
 const withBaseName = makePrefixer("saltOverlayPanelContent");
 
-export const OverlayPanelContent = (props: ComponentPropsWithoutRef<"div">) => {
+export interface OverlayPanelContentProps
+  extends ComponentPropsWithoutRef<"div"> {
+  /**
+   * The content of Overlay Panel
```

Comment:
Should the comments on these be the same?

Reply:
ahh copy and paste :(